### ISSUE TITLE
Morning Session 28/10/2019

### ISSUE THREAD

**Issue by Fig77** [tcc7130/grid-excercise#11]

**Comment by Fig77** [comment]
Honestly I can't say a lot about this since I just read about it last week and haven't put it to practice. Overall looks clean, the only thing in my mind is "don't forget about box model" since a few things (like the header) could've been easier to solve with just a float. But, since this was just to practice flex and grid is not really relevant, is it?

**Comment by Rhelli** [comment]
- If using REM and EM for relative fonts in the rest of your CSS, make sure to set it in the html tag versus your body tag - i'm not sure they quite work the same way.

- Have a look at using the property 'grid-template-areas' when using grid - it makes it a little easier to plan and visualise your grid when your are first laying them out on your main container

- Commenting is a good habit to get into (regardless of whether it is in the specs for our projects) and will be useful for future work in companies. It makes your work more accessible to people appraching your work and makes it maintainable over time (UNLESS your classes and ids are REALLY obvious).

- Indent your classes and ids - most important containers go first and then indent outwards every time you get to a subclass, like so:
![Screenshot 2019-10-28 at 09 48 57](https://user-images.githubusercontent.com/48107460/67688662-30abef00-f968-11e9-92a8-279c782e9396.png)

This makes your code much, much more easy to navigate than flat code and you can find main containers easily.

**Comment by TashfeenRao** [comment]
yeah code is pretty clean & good. i don't have much experience with css. but i can say that your css techniques are good.
